index.html

```diff
@@ -8,18 +8,16 @@
     <link rel="stylesheet" href="index.css" />
   </head>
   <body>
-    <header></header>
+    <header><h1 class="heading">BMI Calculator</h1></header>
     <main>
       <section class="bmi-container">
-        <div class="bmi-box">
-          <h1 class="heading">BMI Calculator</h1>
-        </div>
+        <div class="bmi-box"></div>
```

Comment:
Didn't look into the whole code, just the pr. Why are here a div without content? Do you plan to fill it with content with dynamically with JS?

Reply:
I guess I missed this when moved the ```h1``` . Thx !